bonfire/bonfire.py

```diff
@@ -670,6 +673,20 @@ def _list_namespaces(available, mine, output):
 @click_exception_wrapper("namespace reserve")
 def _cmd_namespace_reserve(name, requester, duration, pool, timeout, local):
     """Reserve an ephemeral namespace"""
+    log.info("Checking for available namespaces to reserve.")
+    if pool_size_limit := get_pool_size_limit(pool):
+        log.info(f"Pool size limit is defined as {pool_size_limit} in '{pool}' pool")
+
+        if get_reserved_namespace_quantity(pool) == get_pool_size_limit(pool):
+            log.info(f"""Namespace max reached for '{pool}' pool
+
+                Max number of namespaces for '{pool}' pool have been reserved. We apologize for
```

Comment:
Can we just keep the error message short, e.g.:
_error(f"maximum number of namespaces for pool `{pool}` (limit: {size_limit}) have been reserved")